src/lib/components/home_page_sections/HomepageHeader/HomepageHeader.svelte

```diff
@@ -6,13 +6,27 @@
 	<WidthContainer>
 		<h1>Welcome to GOV.UK</h1>
 		<p>The best place to find government service and information</p>
-		<p>Simpler, clearer, faster</p>
+		<p><span>Simpler, clearer, faster</span></p>
 	</WidthContainer>
 </header>
 
 <style>
 	header {
 		color: var(--govuk-white);
 		background-color: var(--govuk-blue);
+		padding-top: 60px;
+		padding-bottom: 50px;
+	}
+
+	p {
```

Comment:
I think I'd really like to see some descriptive classes rather than just applying blanket styling to all p tags. Will help to make things clearer if you're debugging later on, also allows you to treat different elements differently. This also applies to the span and h1 below. This will also remove the need for the span itself as you can have `<p class=normal_text>` `<p class=bold_text>` that could even be shared across multiple sections

Reply:
I agree in the fact I should probably add some global classes which I can apply to p tags which all look the same. This will reduce amount of code and make the styles more resuable.

In a svelte file, the css styles applied are only scoped to the component. So using the tag itself to style this component won't cause other p tags in other components to change at least.